SoftLayer/managers/object_storage.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, client):
     def list_accounts(self):
         """Lists your object storage accounts."""
         return self.client.call('Account', 'getHubNetworkStorage',
-                                mask=LIST_ACCOUNTS_MASK)
+                                mask=LIST_ACCOUNTS_MASK, iter=True, limit=10)
```

Comment:
I think the limit should be sent as a method parameter but it should have a default value in case not sent  
 E.g.
          def list_accounts(self, limit=10):

Reply:
Given how slow this api seems to be, having a configurable limit is a good idea.

Reply:
Done.